### PR TITLE
Stop running helix on PRs for now

### PR DESCRIPTION
**Pull request by HaoK**

### CODE REVIEW

**Comment by HaoK** [comment]
Part of https://github.com/aspnet/AspNetCore/issues/8453

**Review by dougbu — APPROVED** [review]

File: .azure/pipelines/helix-test.yml [hunk 1]
```diff
@@ -1,11 +1,7 @@
 # Don't run CI for this config yet. We're not ready to move official builds on to Azure Pipelines
 trigger: none
 
-# Run PR validation on all branches
-pr:
-  branches:
-    include:
-    - '*'
+pr: none # will disable PR builds entirely
```
[the remark this answers is not in the document]
Comment (HaoK) [reply .azure/pipelines/helix-test.yml]: Yeah I noticed that too, for now I just scheduled a daily build against master that runs once a day as a starting point

**Review by jkotalik — CHANGES_REQUESTED** [review]
Can we please discuss this more? This will make my abilities to make any PRs for IIS incredibly hard. There is no other PR validation for IIS besides helix.

**Comment by jkotalik** [comment]
@muratg @shirhatti

**Comment by HaoK** [comment]
Lets use the issue for the discussion

File: .azure/pipelines/helix-test.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkotalik) [reply .azure/pipelines/helix-test.yml]: Can we just narrow down the queue to only include Windows.10.Amd64.EnterpriseRS3.ASPNET.Open listed here: https://github.com/aspnet/AspNetCore/blob/master/eng/targets/Helix.Common.props#L15

[the remark this answers is not in the document]
Comment (jkotalik) [reply .azure/pipelines/helix-test.yml]: This will give half of the IIS coverage (no coverage on IIS Express but it's better than nothing).

**Comment by HaoK** [comment]
Going with https://github.com/aspnet/AspNetCore/pull/8464 instead

**Pull request closed without merging**